tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,3}-ansible{28,29,210}
+envlist = py{27,3}-ansible{29,210,3}
```

Comment:
Dear @btravouillon 

Thanks a lot for this PR 😊 

Would you accept to rebase to current master and change this one to:

`envlist = py27-ansible{29},py3-ansible{29,210,3}`

?

Reply:
I faced issues with automatic tests when running latest versions of Ansible with python2 🙁 
Some j2 variables were not working properly, while they were perfectly working with python3.

Reply:
Do you mean it works with Python 2 + Ansible 2.9 but not with Python 2 + Ansible 2.10?
Can you share any link to the failing test results?

This might be a bug we could fix. [Ansible still supports Python 2](https://docs.ansible.com/ansible/latest/dev_guide/developing_python_3.html#minimum-version-of-python-3-x-and-python-2-x).

Reply:
Yes exactly this is it.

Let me try to find it again. If not, I can do a fake PR that fails.

Reply:
Example : https://github.com/bluebanquise/bluebanquise/runs/2084923975

Reply:
I believe the usage of `vars` in `j2_role_variables` is wrong. I can't find any documentation about this variables in [the documentation about Special Variables](https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html). You should use the [vars lookup](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/vars_lookup.html) instead.

Since 2.10, ansible force usage of OrderedDict with yaml dictionaries for Python 2.7 and < 3.7, which produces the variable below when `j2_roles_variables` uses `vars`:
```
"j2_role_variables": "AnsibleMapping([(u'packages_to_install', [u'dhcp-server']), (u'services_to_start', [u'dhcpd']), (u'firewall_services_to_add', [u'dhcp'])])"
```

With the vars lookup instead, the molecule tests pass. Note that I tried basic lookup. Complex lookups may need additional engineering work to match your implementation.

```
 - name: Converge
   hosts: management1
   vars:
     j2_role_variables: "{{ lookup('vars', role_name + '_' + ansible_facts.os_family + '_' + ansible_facts.distribution_major_version) }}"
```